REFERENCE.md

```diff
@@ -5,7 +5,7 @@
 
 **Classes**
 
-* [`rabbitmq`](#rabbitmq): A module to manage RabbitMQ
+* [`rabbitmq`](#rabbitmq): 
```

Comment:
@ekohl not sure why the `@summary` disappears now? So maybe that leading `# rabbitmq` is necessary?